pyproject.toml

```diff
@@ -1,13 +1,12 @@
-[tool.poetry]
+[project]
```

Comment:
`repository` field is missing. You can check how we did it for another public repository https://github.com/moneymeets/spec2sdk/blob/master/pyproject.toml

Reply:
Ah I see, need to be specified in `tool.poetry` section.